released/packages/coq-linearscan/coq-linearscan.1.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "dev"
+
+homepage: "https://github.com/jwiegley/linearscan"
+dev-repo: "git+https://github.com/jwiegley/linearscan.git"
+bug-reports: "https://github.com/jwiegley/linearscan/issues"
+license: "BSD-3-Clause"
+
+synopsis: "A linear scan register allocator in Coq"
+description: """
+An implementation in Coq of register allocator described in the paper
+Optimized Interval Splitting in a Linear Scan Register Allocator by Christian
+Wimmer and Hanspeter Mӧssenbӧck.
+"""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.14" & < "8.17~") | (= "dev")}
+  "coq-haskell" {(>= "1.1" & < "1.2~") | (= "dev")}
+  "coq-mathcomp-ssreflect" {(>= "1.12" & < "1.15~") | (= "dev")}
```

Comment:
The package works fine on MathComp 1.15.0, so let's increase the bound to support Coq 8.16.0 (MC 1.14.0 doesn't work there).
```suggestion
  "coq-mathcomp-ssreflect" {(>= "1.12" & < "1.16~") | (= "dev")}
```